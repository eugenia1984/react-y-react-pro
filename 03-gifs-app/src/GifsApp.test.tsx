import { describe, test } from 'vitest'
// import { describe, test, expect } from 'vitest'
// import { render } from '@testing-library/react'
// import { GifsApp } from './GifsApp'

describe('GifsApp', () => { 
  test('should render component properly', () => {
    // const { container } = render(<GifsApp />);

    // expect(container).toMatchSnapshot();
  })
})