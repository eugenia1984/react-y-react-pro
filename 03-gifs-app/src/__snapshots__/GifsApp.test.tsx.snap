// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`GifsApp > should render component properly 1`] = `
<div>
  <div
    class="content-center"
  >
    <h1>
      Gifs searcher
    </h1>
    <p>
      Discover and share the perfect GIF
    </p>
  </div>
  <div
    class="search-container"
  >
    <input
      placeholder="Search gifs"
      type="text"
      value=""
    />
    <button>
      Search
    </button>
  </div>
  <div
    class="previous-searches"
  >
    <h2>
      Previous searches
    </h2>
    <ul
      class="previous-searches-list"
    />
  </div>
  <div
    class="gifs-container"
  />
</div>
`;
